### PR TITLE
i18n(es): Update of `api-reference.md`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Update of `api-reference.md` based on this commit da3725bf4e3a24647ee240a3816cc5d72e31d9e6

### CODE REVIEW

File: src/pages/es/reference/api-reference.md [hunk 1]
```diff
@@ -294,45 +294,69 @@ const ip = Astro.clientAddress;
...
 
-| Name           | Type                                              | Description                                        |
-| :------------- | :------------------------------------------------ | :------------------------------------------------- |
-| `has`          | `(name: string) => boolean`                       | Si existe contenido para el nombre del slot          |
-| `render`       | `(name: string, args?: any[]) => Promise<string>` | Representa de forma asíncrona este slot y devuelve HTML   |
+#### `Astro.slots.has()`
+
+**Tipo:** `(slotName: string) => boolean`
+
+Puedes checar si un contenido para un slot específico existe usando `Astro.slots.has()`. Esto puede ser útil cuando quieres envolver el contenido del slot, pero solo quieres renderizar los elementos del envoltorio cuando se está usando el slot.
```
Comment (kevinzunigacuellar) [inline src/pages/es/reference/api-reference.md:301]:
```suggestion
Puedes verificar si el contenido para un slot específico existe usando `Astro.slots.has()`. Esto puede ser útil cuando quieres envolver el contenido del slot, pero solo quieres renderizar los elementos del envoltorio cuando se está usando el slot.
```

File: src/pages/es/reference/api-reference.md [hunk 2]
```diff
@@ -294,45 +294,69 @@ const ip = Astro.clientAddress;
...
 <Fragment set:html={html} />
 ```
-<!-- Waiting for bug fix from Nate; reformat CAREFULLY when un-uncommenting out!
 
+:::note
+Esta nota es para casos de uso avanzados! En la mayoría de los casos, es más simple renderizar el contenido del slot con [el elemento `<slot />`](/es/core-concepts/astro-components/#slots).
+:::
 
-`Astro.slots.render` optionally accepts a second argument, an array of parameters that will be forwarded to any function children. This is extremely useful for custom utility components.
+`Astro.slots.render()` opcionalmente acepta un segundo argumento: una array de parámetros que se reenviarán a cualquier función de niños. Esto puede ser útil para componentes de utilidad personalizados.
```
Comment (kevinzunigacuellar) [inline src/pages/es/reference/api-reference.md:333]:
```suggestion
`Astro.slots.render()` opcionalmente acepta un segundo argumento: una array de parámetros que se enviarán a cualquier función hija. Esto puede ser útil para componentes de utilidad personalizados.
```

File: src/pages/es/reference/api-reference.md [hunk 3]
```diff
@@ -294,45 +294,69 @@ const ip = Astro.clientAddress;
...
 
-Given the following `Message.astro` component...
+Port ejemplo, este componente `<Shout />` convierte su prop `message` en mayúsculas y la pasa al slot predeterminado:
```
Comment (kevinzunigacuellar) [inline src/pages/es/reference/api-reference.md:335]:
```suggestion
Por ejemplo, este componente `<Shout />` convierte su prop `message` en mayúsculas y la pasa al slot predeterminado:
```

**Comment by dreyfus92** [comment]
Thanks @kevinzunigacuellar

**Pull request merged**